fenix/app/src/main/java/org/mozilla/fenix/shopping/middleware/ReviewQualityCheckTelemetryMiddleware.kt

```diff
@@ -117,4 +132,16 @@ class ReviewQualityCheckTelemetryMiddleware : ReviewQualityCheckMiddleware {
             }
         }
     }
+
+    private fun isStaleAnalysis(
+        state: ReviewQualityCheckState,
+    ): Boolean =
```

Comment:
Since we are acting on the state in this function, we could opt to make it an extension function on `ReviewQualityCheckState`